apps/percona/chaos/node_freeze/run_litmus_test.yml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: node-freez-
+  namespace: litmus 
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        freezer: node-mem
+
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openenbs/ansible-runner:ci
+        imagePullPolicy: Always
```

Comment:
IMO better to have a node-selector field. It will be useful in the future(infra chaos pipeline)